packages/wrangler/src/api/startDevWorker/types.ts

```diff
@@ -161,6 +162,7 @@ export interface StartDevWorkerInput {
 		enableServiceEnvironments?: boolean;
 	};
 	unsafe?: Omit<CfUnsafe, "bindings">;
+	experimentalAssets?: ExperimentalAssets;
```

Comment:
startWorker's types moved away from conflating resource + binding config. This property should only contain the directory. And the `Binding` union type below should get an additional member: `{ type: 'assets' }`

Reply:
Can you also move this to `experimental.assets` instead please

Reply:
✅